stac_fastapi/types/setup.py

```diff
@@ -6,9 +6,7 @@
     desc = f.read()
 
 install_requires = [
-    # We need to pin fastapi for the openapi schema
-    # See https://github.com/stac-utils/stac-fastapi/issues/242
-    "fastapi==0.67.*",
+    "fastapi>=0.67,!=0.68.0,!=0.68.1,!=0.68.2,!=0.69.0,!=0.70.0,!=0.70.1,!=0.71.0,!=0.72.0",
```

Comment:
Why not just set this to `fastapi>=0.73`?

Reply:
To avoid setting a breaking change. Basically setting it as it is right now keeps the compatibility with old version 🤷‍♀️

Reply:
Sorry I'm slow to respond 😬 .  Because the range bridges several major versions I'd rather just commit to the `0.73.*` version instead of allowing both `0.67.*`and `0.73.*`

Reply:
fine by me!